testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/hibernate/classfiletransformertest/ClassFileTransformerTestCase.java

```diff
@@ -91,10 +92,10 @@ public void testhibernate_ejb_use_class_enhancer() throws Exception {
     }
 
     @Test
-    public void testHibernateByteCodeEnhancement() {
+    public void testHibernateByteCodeEnhancementIsDisabledByDefault() {
         // Note: ManagedTypeHelper is an internal Hibernate ORM class, if it is removed or renamed then this test can be updated
         // accordingly.
-        assertTrue("Employee class is not bytecode enhanced", org.hibernate.engine.internal.ManagedTypeHelper.isManagedType(Employee.class));
+        assertFalse("Employee class is not bytecode enhanced", org.hibernate.engine.internal.ManagedTypeHelper.isManagedType(Employee.class));
```

Comment:
Should this be "Employee class is bytecode enhanced" if it's disable by default and we want to ensure the bytecode enhancing has not been done?